docs/src/SUMMARY.md

```diff
@@ -10,6 +10,7 @@
     - [Execution contexts](./user_docs/assembly/execution_contexts.md)
     - [Flow Control](./user_docs/assembly/flow_control.md)
     - [Field Operations](./user_docs/assembly/field_operations.md)
+    - [Extension Field Operations](./user_docs/assembly/extension_field_operations.md)
```

Comment:
I would probably put these under "Field Operations" - just in a separate section there.